components/form/GithubPicker.vue

```diff
@@ -303,13 +303,13 @@ export default {
           </template>
           <template slot="option" slot-scope="option">
             <div class="d-center">
-              <img :src="option.owner.avatar_url" />
+              <img :src="option.owner.avatar_url" alt="altText" />
```

Comment:
This sets the alt text to 'altText' - I don't think that's what we would want - we should use localised strings.

Reply:
translations added, please review